plugins/woocommerce/tests/e2e-pw/tests/activate-and-setup/complete-onboarding-wizard.spec.js

```diff
@@ -3,13 +3,9 @@ const { onboarding } = require( '../../utils' );
 const { storeDetails } = require( '../../test-data/data' );
 const { api } = require( '../../utils' );
 const { features } = require( '../../utils' );
+const { describe } = require('node:test');
 
-// Skipping Onbaording tests when the core-profiler is enabled.
-const testRunner = features.is_enabled( 'core-profiler' )
-	? test.describe.skip
-	: test.describe;
-
-testRunner( 'Store owner can complete onboarding wizard', () => {
+test.skip( 'Store owner can complete onboarding wizard', () => {
```

Comment:
Everything looks good to me, I'm just wondering if these tests should actually be removed rather than perpetually skipped?

Reply:
👍 

Good question! Yes, we should remove them. I'll create an issue to follow up. We'll need to write E2E tests for the core profiler.